app/books.util.js

```diff
@@ -34,23 +48,42 @@ export default class BookAPIUtil {
   /*
     https://developers.google.com/books/docs/v1/reference/volumes/list
     Return Type : Array[JSON]
-    [
-      {
-        googleBookId: String,
-        title: String,
-        authors: Array[String],
-        publisher: String,
-        publishedDate: String,
-        categories: Array[String],
-        // optional
-        isbn13: String,
-        thumbnail: String
-      }
-    ]
+    {
+      pageInfo: {
+        totalSize: String,   // totalSize can be changed, even if you search same text.
+        startIndex: Number,
+        lastIndex: Number,
+        hasNext: Boolean,
+        hasPrev: Boolean
+      },
+      bookList: [
+        {
+          googleBookId: String,
+          title: String,
+          authors: Array[String],
+          publisher: String,
+          publishedDate: String,
+          categories: Array[String],
+          // optional
+          isbn13: String,
+          thumbnail: String
+        }
+      ]
+    }
   */
 
-  static searchBooks(keyword) {
-    const address = `${GOOGLE_BOOKS_API.URL}?q=${keyword}&key=${GOOGLE_BOOKS_API.API_KEY}`;
+  static searchBooks(keyword, { orderBy = 'relevance', maxResults = 5, startIndex = 0, printType = 'books' }) {
+    const options = new Map()
+      .set('orderBy', orderBy)
+      .set('maxResults', maxResults)
+      .set('startIndex', startIndex)
+      .set('printType', printType);
+
+    let query = '';
+    options.forEach((value, key) => {
```

Comment:
using for loop is preferred. here are some ref: http://stackoverflow.com/questions/22155280/why-is-native-javascript-array-foreach-method-significantly-slower-than-the-stan
and we can use it like below:
```jsx
for (let [value, key] of options) {
  query += `&${key}=${value}`;
}
```